emulator_test.go

```diff
@@ -136,6 +137,23 @@ func TestTemperatureEmulationAnomalies_Trend(t *testing.T) {
 	assert.True(t, mean(results) > emulator.T.MeanTemperature)
 }
 
+func TestTemperatureEmulationAnomalies_Decrease_Trend(t *testing.T) {
+	emulator := createEmulator(14400, 0)
+	emulator.T.IsTrendAnomaly = true
+	emulator.T.TrendAnomalyMagnitude = 30.0
+	emulator.T.TrendAnomalyLength = 1e3
+	emulator.T.IsRisingTrendAnomaly = false
+	step := 0
+	var results []float64
+	for step < 1e4 {
+		emulator.Step()
+		results = append(results, emulator.T.T)
+		step += 1
+	}
+
+	assert.True(t, mean(results) < emulator.T.MeanTemperature)
```

Comment:
Is there a more specific assertion we could introduce here? Perhaps ensuring it is much less than `T.MeanTemperature`?

Reply:
Ignored, original implementation was fine